src/pudl/metadata/fields.py

```diff
@@ -1341,6 +1402,24 @@
         ),
         "type": "string",
     },
+    "flue_gas_bypass_fgd": {
+        "type": "boolean",
+        "description": "Indicates whether flue gas can bypass the flue gas desulfurization unit.",
+    },
+    "flue_gas_entering_fgd_pct_of_total": {
```

Comment:
Not for this PR, but we should maybe make clear when a column is expected to be a ratio (0-1) and when it's expected to be a percent.

Then users can use that explicit signal instead of "are all the values between 0 and 1? then probably it's a ratio."

Two ways I could imagine doing this are:
- naming columns with `_ratio` instead of `pct_of_total`
- using the `"unit"` to clarify "percent" vs. "ratio"